config/routes.rb

```diff
@@ -14,6 +14,8 @@
 
   namespace :api do
     namespace :v1 do
+      resource 'login', only: [:create]
```

Comment:
make this a single `post` route. we're not considering `login` a resource in our code, yet.